examples/text-generation/quantization_config/maxabs_measure.json

```diff
@@ -2,8 +2,6 @@
     "method": "HOOKS",
     "mode": "MEASURE",
     "observer": "maxabs",
-    "allowlist": {"types": [], "names":  []},
-    "blocklist": {"types": [], "names":  []},
     "dump_stats_path": "./hqt_output/measure",
     "dump_stats_xlsx_path": "./hqt_output/measure/fp8stats.xlsx"
```

Comment:
sorry for not mentioning it before, but would be great if you could remove this line too. from all files.

Reply:
do you mean the `dump_stats_path` line or `dump_stats_xlsx_path` line?
your comment is under the latter but only the former appears in other files.

Reply:
right, I meant we should remove: dump_stats_xlsx_path
thanks :)

Reply:
Done